<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BingeWatchClass</class>
 <widget class="QMainWindow" name="BingeWatchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>追剧神器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{
	width: 80px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBZhan">
       <attribute name="title">
        <string>B站</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="lstBZhan">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget::item:selected { 
	color: rgb(255, 255, 255);
	background-color: rgb(31, 136, 61);
 }
QListWidget::item{
	height: 30px;
}</string>
          </property>
          <property name="currentRow">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>查看新内容</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </item>
          <item>
           <property name="text">
            <string>监控更新</string>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stkWgtBZhan">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pgBZhanView">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tbBZhanView">
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>用户</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>视频名称</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>时长</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>URL</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pgBZhanMonitor"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDouYin">
       <attribute name="title">
        <string>抖音</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTabBar::tab{
	width: 80px;
	height: 25px;
}
QTabBar::tab:selected{
	background-color: rgb(255, 170, 127);
}</string>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabDouYinView">
           <attribute name="title">
            <string>查看</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabDouYinMonitor">
           <attribute name="title">
            <string>监控</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabKuaiShou">
       <attribute name="title">
        <string>快手</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BingeWatch.qrc"/>
 </resources>
 <connections/>
</ui>
